src/components/KLabel/KLabel.vue

```diff
@@ -1,11 +1,14 @@
 <template>
   <label
+    :id="labelId"
     class="k-label"
     :class="{ 'required': required }"
   >
     <slot />
+
     <KTooltip
       v-if="hasTooltip"
+      :aria-labelledby="labelId"
```

Comment:
question: I'm not sure this makes sense in the context?

You have a label, which essentially provides info about another element, e.g. input.

This is then saying that the KTooltip is "labeled by the label"

Reply:
Yes, it's providing accessible association between the tooltip and label. Otherwise it's impossible for screen readers to know the tooltip is somehow related to the label.

Reply:
What I mean is I think the relationship here is backwards - the tooltip provides extra info about the label, not the other way around

Reply:
Yes, I get what you are saying but `aria-labelledby` needs to be set on an interactive elements - which in this case is the tooltip. Doing the other way around (setting it on label element) would be incorrect.

Reply:
But isn't saying "the tooltip is described by the label" incorrect? 

The tooltip is offering additional information within the label itself, about the label content

Reply:
Labelled by, not described by - those are 2 different attributes. And that actually might be a good solution here - to set `aria-describedby` on label element to point at tooltip content.. only thing I'm not sure about is whether the two are interchangeable (if we did add `aria-describedby` on label would we still need to keep `aria-labelledby` on the tooltip). Let me do some reading and get back to you.

Reply:
You should be able to drop labelledby; aria-attributes are additive, meaning since the Tooltip is contained within the label, it's inferred

Reply:
Okay, updated now